test/units/module_utils/basic/test_tmpdir.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015-2018 Ansible Project
```

Comment:
Not a blocker: You can just use the year 2018 since this is a new file.

Reply:
Cool, I just copied and pasted from another file and changed the end date, good to know you can just set the year.